beast/physicsmodel/creategrid.py

```diff
@@ -387,6 +406,10 @@ def make_extinguished_grid(spec_grid, filter_names, extLaw, avs, rvs,
         if npts == 0:
             raise AttributeError('No valid points')
     else:
+        dustpriors = PriorWeightsDust(avs, av_prior_model,
+                                      rvs, rv_prior_model,
+                                      [1.0], {'name': 'flat'})
```

Comment:
Replace `{'name': 'flat'}` here with `fA_prior_model` which is set to that dictionary value as default keyword.